spec/models/notification_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe Notification, type: :model do
+  let!(:super_admin) { create(:user, :super_admin)}
+  let!(:organization) { create(:organization) }
+  let!(:admin) { create(:user, :admin, organization_id: organization.id) }
+  let!(:manager) { create(:user, :manager, organization_id: organization.id) }
+  # let!(:notification) { create(:notification, :user_added, notificable: manager, user: admin) }
+
+  describe 'factory specs' do
+    it 'has factory' do
+      expect(create(:notification)).to be_persisted
```

Comment:
fyi: FactoryBot raises error if it can't create record using `create` helper!
completely useless test